cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/ACLineSegmentConversion.java

```diff
@@ -165,7 +166,11 @@ private void convertDanglingLine(int boundarySide) {
                         .setVoltageRegulationOn(false)
                     .add()
                     .add();
+            addAliases(dl);
         }
+        dl.setProperty("boundarySide", String.valueOf(boundarySide));
```

Comment:
If you have a lot of properties, I think that an extension is better for dangling line.